SubEthaEdit-Mac/Source/PlainTextEditor.m

```diff
@@ -2157,11 +2157,11 @@ - (void)keyDown:(NSEvent *)aEvent {
                     break;
                 }
             }
-            NSRect frame = [self.O_editorView frame];
-            frame.size.width = 50;
-            frame.origin.y = frame.size.height - 20;
-            frame.size.height = 20;
-            [s_cell performClickWithFrame:frame inView:self.O_editorView];
+
+            NSView * controlView = self.topBarViewController.view;
```

Comment:
Coding style for this is no space after star please. E.g. `NSView *controlView…`